.github/workflows/_triage.yaml

```diff
@@ -85,6 +85,10 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Skopeo Login to GitHub Container Registry
```

Comment:
Turns out Skopeo uses docker CLI under the hood. So the previous `docker/login-action` step already suffices and we don't need `skopeo login` for accessing ghcr.io images.

Reply:
Removed it in da30162